pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md

```diff
@@ -0,0 +1,528 @@
+---
+$title: Actions and events
+order: 7
```

Comment:
For my own learning, what does `order: 7` mean?

Reply:
Sure! This is the order the document appears in the sidebar under "Learn" - which appears to be inaccurate...